pkg/storage/stores/shipper/bloomshipper/config/config.go

```diff
@@ -8,11 +8,23 @@ import (
 )
 
 type Config struct {
-	WorkingDirectory string `yaml:"working_directory"`
+	WorkingDirectory       string                 `yaml:"working_directory"`
+	BlocksDownloadingQueue DownloadingQueueConfig `yaml:"blocks_downloading_queue"`
```

Comment:
Do we really need the nesting here? Leave that up to you.

Reply:
I believe that it will be better to have it separated ;) also do not have a strong opinion on it... just separated it because in the next PR I will add cache config and do not want to mix the properties in the single block....